blacklight-cornell/app/views/catalog/_call_number_browse_links.html.erb

```diff
@@ -42,7 +42,9 @@
 	%>
 	<div class="">
 	  <div id="classification" class="row class-heading" data-anchor-label="<%= internal_class%>" data-keep-count="true" data-next-count="0" data-prev-count="0">
-        <div class="vb-browse-link"><a href="/browse?authq=<%=callnumbers[0]%>&browse_type=virtual" onclick="javascript:_paq.push(['trackEvent', 'virtualbrowse_itemview', 'expanded_view_link']);">Expanded view</a></div>
+        <div class="vb-browse-link">
+	  		<%= link_to "Expanded view", browse_url(callnumbers[0], "virtual"), {:onclick => "javascript:_paq.push(['trackEvent', 'virtualbrowse_itemview', 'expanded_view_link']);"} %>
```

Comment:
I would recommend using rails route helpers as a more rails-y way of generating these urls, with built-in parameter encoding: https://guides.rubyonrails.org/routing.html#naming-routes

So instead of creating a custom helper method, I would do: `browse_index_path(authq: callnumbers[0], browse_type: 'virtual')`.

Reply:
*_path comes from that rails route helper and browse_index_* comes from the named route in config/routes.rb

Reply:
ok, will do. thanks for the suggestion!